ui/src/lib/gql/queries/workspace.ts

```diff
@@ -0,0 +1,114 @@
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+
+import { CreateWorkspaceMutation, useGraphQLContext } from "@flow/lib/gql";
+
+import { graphql } from "../__gen__";
+
+graphql(`
+  mutation CreateWorkspace($input: CreateWorkspaceInput!) {
+    createWorkspace(input: $input) {
+      workspace {
+        id
+      }
+    }
+  }
+`);
+
+// TODO: Should this a fragment in GET_ME?
```

Comment:
I think having a Workspace fragment would be helpful here. Then you can use it in all of these queries and mutations so you can consistently get the same object back and update them all at once if the backend api changes

Reply:
Cool. This makes sense to me :+1: